README.md

```diff
@@ -81,83 +95,25 @@ GET https://infra.devskills.app/api/accounting/accounts/0afd02d3-6c59-46e7-b7bc-
 
 </details>
 
-## Additional requirements
+### Solution expectations
 
 - Do your best to make the [provided E2E tests](cypress/e2e/test.cy.js) pass.
 
 <!--TASK_INSTRUCTIONS_END-->
 
-## Getting started
-
-<details>
-  <summary>If you run into a problem</summary>
-  
-1. **Open a [GitHub Issue](https:\/\/docs.github.com\/en\/issues\/tracking-your-work-with-issues\/creating-an-issue):** Simply go to the "Issues" tab in this repository and click on "New issue".
-2. **Describe Your Issue:** Briefly describe the problem you are encountering. Include key details like error messages or steps to reproduce the issue. This helps us understand and resolve your concern more efficiently.
-3. **Automated Support:** Initially, our support bot will try to resolve your issue. If it is unable to help, a member of the Alva team will be notified and will step in to assist you.
-
-**Note:** it is important to close the issue once your problem is resolved, open issues may indicate to the hiring team that your assignment is not yet ready for review.
+### Running the E2E tests
```

Comment:
should this be covered by this article? https://help.alvalabs.io/en/articles/9028831-how-to-work-with-cypress-tests